### PR TITLE
Add flag description for Paddle-TensorRT

### PR DESCRIPTION
**Pull request by jeng1220**
Many users don't know how to build Paddle-TensorRT due to lacking description, so add needed information

Preview：

image

### CODE REVIEW

**Comment by jeng1220** [comment]
@NHZlX 
Please take a look

File: doc/fluid/beginners_guide/install/Tables.md [hunk 1]
```diff
@@ -174,12 +174,18 @@
...
+	<tr>
 		<tr>
 		<td> CUDA_ARCH_NAME </td>
 		<td> 是否只针对当前CUDA架构编译 </td>
 		<td> All:编译所有可支持的CUDA架构 可选：Auto 自动识别当前环境的架构编译 </td>
 	</tr>
+	<tr>
+		<tr>
+		<td> TENSORRT_ROOT </td>
+		<td> 指定TensorRT路徑 </td>
```
Comment (NHZlX) [inline doc/fluid/beginners_guide/install/Tables.md]: 这里换成简体吧

File: doc/fluid/beginners_guide/install/Tables.md [hunk 2]
```diff
@@ -174,12 +174,18 @@
...
+		<td> 不指定會自動搜尋TensorRT </td>
```
Comment (NHZlX) [inline doc/fluid/beginners_guide/install/Tables.md]: 这里代表是默认值。 “不指定會自動搜尋TensorRT ” -> “windows下默认值为'/'， Linux下默认值为 '/usr/' ”

**Comment by jeng1220** [comment]
@NHZlX ,
I have modified the description

**Review by NHZlX — APPROVED** [review]
LGTM

**Review by saxon-zh — APPROVED** [review]

**Pull request merged**